pytest.ini

```diff
@@ -13,12 +13,11 @@ python_files = test*.py
 # http://doc.pytest.org/en/latest/writing_plugins.html#collection-hooks
 
 # Both our plugin and unittest provide their own collection logic,
-# So we can disable the default python collector by giving it empty
+# so we can disable the default python collector by giving it empty
 # patterns to search for.
 # Note that unittest requires that no "Test*" classes exist.
 python_classes =
 python_functions =
 
 # always run in parallel (requires pytest-xdist, see test-requirements.txt)
-addopts = -nauto
-
+addopts = -nauto -ra
```

Comment:
I am also not a fan of `-n auto` here really, which gets in your way when running a single test etc (but `-n0` can be used then, so it is not too bad).

Reply:
Yeah, we have `-n0` in the developer docs IIRC, it is just more often that you want to run with `-n auto`, so that is why it is the config.